src/data/causes/blackEquity.json

```diff
@@ -0,0 +1,116 @@
+    },
+    "sections": {
+      "landing": {
+        "title": "Support Black equity, every day",
+        "subtitle": "Every tab opened raises money for nonprofit organizations working toward Black empowerment and systemic reform.",
+        "ctaImg": "../../img/blackEquity/landing.png",
+        "waveColor": "#A27DF8"
+      },
+      "moneyRaised": {
+        "moneyImg": "../../img/blackEquity/moneyRaised.png"
+      },
+      "charityIntro": {
+        "title": "Stop wasting tabs, start changing the world.",
+        "subTitle": "Little actions can have big results. With Tab for Black Equity, you can help make a positive impact on charities uplifting Black communities just by opening more browser tabs.",
+        "steps": [
+          {
+            "img": "../../img/blackEquity/charityIntro1.png",
+            "text": "More young people going to college"
+          },
+          {
+            "img": "../../img/blackEquity/charityIntro2.png",
+            "text": "Bail money for people who wouldn't otherwise have it"
+          },
+          {
+            "img": "../../img/blackEquity/charityIntro3.png",
+            "text": "A better, more equitable planet"
+          }
+        ],
+        "waveColor": "#A27DF8"
+      },
+      "TFACIntro": {
+        "titleImg": "../../img/blackEquity/introCTA.png",
+        "title": "Make an impact towards Black equity.",
+        "subtitle": "Turn your daily habits into a positive change. Do good, feel good.",
+        "img1": "../../img/blackEquity/intro1.png",
+        "img1Subtext": "##### Open new tabs on your browser.",
+        "img2": "../../img/blackEquity/intro2.png",
+        "img2Subtext": "##### Earn money for charities.",
+        "img3": "../../img/blackEquity/intro3.png",
+        "img3Subtext": "##### Help support systemic change and Black empowerment."
+      },
+
+      "Mission": {
+        "missionURL": "/black-equity/",
+        "titleText": "Our Mission",
+        "subtitleText": "To make giving back easier than ever",
+        "bodyText": "We believe everyone should make a difference for racial equity. Through our partnerships with Thurgood Marshall College Fund and the Bail Project, you can be sure each tab you open is advancing Black equity.",
+        "image": "../../img/blackEquity/mission.png"
+      },
+      "Security": {
+        "titleImg": "../../img/blackEquity/security.png"
+      },
+      "Financials": {
+        "title": "Transparent Financials",
+        "text": "Every tab does make an impact!  Check out how much we've raised together.",
+        "buttonText": "See All Financials",
+        "ctaImg": "../../img/blackEquity/financials.png",
+        "q1Img": "../../img/cats/q1.png",
+        "q2Img": "../../img/cats/q2.png",
+        "q3Img": "../../img/cats/q3.png",
+        "q4Img": "../../img/cats/q4.png"
```

Comment:
Need the globe images for these:
https://www.figma.com/file/WwF1hgTbUCCVNiT7dA5TM1/TF-Black-Equity?node-id=0%3A23

These will be the generic doodles. TeamSeas will use them too.